src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1710,22 +1717,6 @@ void TableDockWidget::writeMascotGeneric(QString filename) {
   file.close();
 }
 
-void TableDockWidget::align() {
```

Comment:
Could you please tell me why these functions are removed? Were these not being used or these were the part of that galleryWidget that is not being used now

Reply:
No this was called from and within TableDockWidget itself. Maybe this was some old code that used to work, but it no longer makes sense to me. we are not going to do selective alignment based on just a few groups.

Reply:
okay. got it